maat-court-data-api/src/main/java/gov/uk/courtdata/repository/ContributionFilesRepository.java

```diff
@@ -0,0 +1,21 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.ContributionFilesEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Repository
+public interface ContributionFilesRepository extends JpaRepository<ContributionFilesEntity, Integer> {
+
+    @Query(value = "SELECT cf.xml_content FROM TOGDATA.CONTRIBUTION_FILES CF WHERE CF.FILE_NAME like '%%CONTRIBUTIONS%%' AND CF.DATE_CREATED BETWEEN fromDate and toDate",
```

Comment:
This query is not going to be very performant as the `DATE_CREATED` column has no index defined on it and the only other search criteria uses regex (%%).

Can we have the table definition updated to include an index on the `DATE_CREATED` column?

Reply:
Yes, the query could improve tho. I was also thinking of parametrising the 'like' part of the query. 

However, we need to determine the impact of updating the table def (it should be just fine) on other parts of the systems (like the hub job etc.). We should be fine for now and will monitor the performance, and then we could look into optimising the query.